src/odemis/acq/stream/_live.py

```diff
@@ -425,8 +437,11 @@ def guessFoV(self):
             # control it from Odemis. In this case, the user has to type the
             # current magnification in, which directly sets the pixelSize.
             # The stream never has a local version of any of them.
-            base_pxs = self.emitter.pixelSize.value  # It's read-only and always present on the emitter
-            hpxs = base_pxs[0] * scale[0]
+            if hasVA(self.emitter, "pixelSize"):
+                base_pxs = self.emitter.pixelSize.value  # It's read-only and present on almost every emitter
+                hpxs = base_pxs[0] * scale[0]
+            else:
+                raise AttributeError("Failed to estimate the field-of-view, the emitter has no pixelSize VA.")
```

Comment:
@pieleric I have added the things to deal with guessFoV in a separate commit. These lines are basically the only change. After that we could merge it. (I haven't tested it on Helios yet but I don't think that is necessary)